Plugins/GitFlow/GitFlowForm.cs

```diff
@@ -149,7 +149,7 @@ private IReadOnlyList<string> GetBranches(string typeBranch)
             GitArgumentBuilder args = new("flow") { typeBranch };
             var result = _gitUiCommands.GitModule.GitExecutable.Execute(args);
 
-            if (result.ExitCode != 0 || result.StandardOutput is null)
+            if (!result.ExitedSuccessfully || result.StandardOutput is null)
```

Comment:
Can `result.StandardOutput is null` be ever true?

Reply:
The implementations of various `CleanString()` in `ExecutableExtensions` is not annotated to be null, so I guess that can me changed too.